### PR TITLE
Add workflow to build the package and upload to dppy/label/dev

### PR DESCRIPTION
**Pull request by oleksandr-pavlyk**
Add workflow to build the package and upload to dppy/label/dev

Fix post-link script so that tests pass.

Add SECURTY.md and CODEOWNERS to improve OpenSSF score.

### CODE REVIEW

**Comment by oleksandr-pavlyk** [comment]
Once this package is built and uploaded, activation scripts in `dpctl` which perform similar function should be removed.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*  @oleksandr-pavlyk @xaleryb @ZzEeKkAa @ndgrigorian
```
Comment (ndgrigorian) [inline .github/CODEOWNERS:1]: The CODEOWNERS file requires @ZzEeKkAa and I have write access, otherwise Github says it's incorrect

**Comment by ZzEeKkAa** [comment]
It's great package, but have you consider to port it to `conda-forge` directly?

**Comment by ZzEeKkAa** [comment]
And if we are maintaining feedstock here, is it possible to use `conda-smithy` to reuse all the things conda-forge have?
https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks

**Review by xaleryb — APPROVED** [review]

**Comment by oleksandr-pavlyk** [comment]
@ZzEeKkAa This package is `noarch` and python version agnostic, see `noarch: generic` in the build section of "meta.yaml". Setting up conda-forge-like build infrastructure to build it seems an overkill.

**Review by ndgrigorian — APPROVED** [review]

**Pull request merged**